cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -72,105 +72,104 @@
-                {% elif input.type == 'email'
-                   or input.type == 'number'
-                   or input.type == 'password'
-                   or input.type == 'tel'
-                   or input.type == 'text'
-                   or input.type == 'url' %}
-
-                    <input type="{{ input.type }}"
-                           value="{{ input.attrs.value if input.attrs.value else '' }}"
-                           placeholder="{{ input.attrs.placeholder if input.attrs.placeholder else '' }}"
-                           {{ 'id=filter_' ~ input.id if input.id else '' }}
-                           {{ 'name=filter_' ~ input.name if input.name else '' }}>
-                {% elif input.type == 'checkbox' %}
-                    {% for checkbox in input.attrs -%}
-                        {{ _filter_checkbox(checkbox.label,
-                                            checkbox.id,
-                                            checkbox.name,
-                                            checkbox.value) | safe }}
-                    {% endfor %}
-                {% elif input.type == 'select' %}
-                    <select {{ 'id=filter_' ~ input.id if input.id else '' }}
-                            {{ 'name=filter_' ~ input.name if input.name else '' }}
-                            {{ 'multiple' if input.multiple else '' }}
-                            placeholder="{{ input.attrs.placeholder if input.attrs and input.attrs.placeholder else '' }}">
-                    {% for option in input.opts -%}
-                        {{ _filter_option(option.label, option.value) | safe }}
-                    {% endfor %}
-                    </select>
-                {% endif %}
-            </div>
-        </div>
-    {% endfor %}
-{%- endmacro %}
-
-{% macro _filter_checkbox(label_text, id, name, value) %}
+{% macro _filter_selectable(type, index, label_text, id, name, value) %}
     <input class="cf-input"
-           type="checkbox"
+           type="{{ type }}"
            value="{{ value }}"
-           id="filter_{{ id }}"
-           name="filter_{{ name }}"
-           {{ 'checked' if is_filter_selected(name, value) else '' }}>
-    <label for="filter_{{ id }}" >
+           id="filter{{ index }}_{{ id }}"
+           name="filter{{ index }}_{{ name }}"
+           {{ 'checked' if is_filter_selected('filter' ~ index ~ '_' ~ name, value) else '' }}>
+    <label for="filter{{ index }}_{{ id }}" >
         {{ label_text if label_text else value }}
     </label>
 {% endmacro %}
 
 {% macro _filter_option(label_text, value) %}
     <option value="{{ value }}"
-            {{ 'selected' if is_filter_selected(label_text, value) else '' }}
-            >
+            {{ 'selected' if is_filter_selected(label_text, value) else '' }}>
         {{ label_text }}
     </option>
 {% endmacro %}
 
+{% if not page %}
+    {% macro _prototype_filter_form(value) %}
+        {% if value.action and value.action != '/' %}
```

Comment:
Totally, but since we're removing prototype code I didn't see a need to change it. I'm removing it before merging.